db/collab.go

```diff
@@ -0,0 +1,58 @@
+package db
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/labstack/echo/v4"
+	"github.com/uclaacm/teach-la-go-backend/httpext"
+	"golang.org/x/net/websocket"
+)
+
+// Session describes a collaborative coding environment.
+type Session struct {
+	// Map UIDs to their websocket.Conn
+	Conns map[string]*websocket.Conn `json:"conns"`
```

Comment:
Just looking for some input, do you think that this should be json annotated? In retrospect, I don't know how useful it might be.

Reply:
I'll remove it; we can always add it back